mana: Microsoft Azure Network Adapter (MANA) RDMA provider

longlimsft opened the pull request:
Introduce a provider that exposes MANA devices to user applications.

longlimsft commented:
Hi,

Kernel patch is being reviewed at v4:
https://lore.kernel.org/all/1655345240-26411-1-git-send-email-longli@linuxonhyperv.com/T/#m50c39d647bc41f3641ca491b72b82aa225b81d66

Thanks,
Long

jgunthorpe commented:
Please rebase it

The first commit must be generated with the kernel-headers/update script. Use commit ID 0266a177631d4c6b963b5b12dd986a8c5abdbf06

jgunthorpe commented on providers/mana/man/CMakeLists.txt:
```diff
@@ -0,0 +1,5 @@
+rdma_man_pages(
+  manadv.7.md
+  manadv_set_context_attr.3.md
+  manadv_init_obj.3.md
```
Keep sorted

longlimsft replied:
Fixed, thank you.

rleon commented:
I don't see any progress here. It is not late to revert mana driver from upcoming PR to Linus.

longlimsft commented:
Sorry, I was on vacation with limited access to lab.

I’m starting to work on this today.

________________________________
From: Leon Romanovsky ***@***.***>
Sent: Thursday, December 1, 2022 1:28 AM
To: linux-rdma/rdma-core ***@***.***>
Cc: Long Li ***@***.***>; Author ***@***.***>
Subject: Re: [linux-rdma/rdma-core] mana: Microsoft Azure Network Adapter (MANA) RDMA provider (PR #1177)


I don't see any progress here. It is not late to revert mana driver from upcoming PR to Linus.

—
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Flinux-rdma%2Frdma-core%2Fpull%2F1177%23issuecomment-1333474417&data=05%7C01%7Clongli%40microsoft.com%7Cc22794eb4cf54364122908dad37e6fee%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638054837246747008%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=wZ5Wv%2BMMQbogPAfl2D%2BDtIh40xpa%2B28hd8KnQBBtWvY%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAAUR3S2HR25ZRPZPOL6Z62DWLBVUVANCNFSM5WYYRI5A&data=05%7C01%7Clongli%40microsoft.com%7Cc22794eb4cf54364122908dad37e6fee%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638054837246747008%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=E5qcy3pVAhNqgajq%2BUHvcCt%2Fpk4KvQvDb2pEXoia7nY%3D&reserved=0>.
You are receiving this because you authored the thread.Message ID: ***@***.***>

longlimsft commented:
> Please rebase it
> 
> The first commit must be generated with the kernel-headers/update script. Use commit ID 0266a177631d4c6b963b5b12dd986a8c5abdbf06

I have rebased the pull request. The first commit generated by kernel-headers/update script doesn't include an entry in rdma_kernel_provider_abi() in CMakeLists.txt, so I added the following with this commit:

@@ -70,6 +71,7 @@ rdma_kernel_provider_abi(
   rdma/hns-abi.h
   rdma/ib_user_verbs.h
   rdma/irdma-abi.h
+  rdma/mana-abi.h
   rdma/mlx4-abi.h
   rdma/mlx5-abi.h
   rdma/mthca-abi.h

Please let me know if this should be done in some other way.
